### DISCUSSION TITLE
[Cache] Redis connection failed: Redis server went away

### DISCUSSION THREAD

**Discussion by polina-suzdal** [polina-suzdal/symfony-discusion#6] [Q&A] [answered]
Symfony version(s) affected
6.2.3

Description
https://github.com/symfony/symfony/pull/48711 breaks symfony/cache with Redis at least for me

How to reproduce
Update symfony from 6.2.2 to 6.2.3

I have set this REDIS_PROVIDER=redis://localhost/2 in my .env, after Upgrade to 6.2.3 i get the error:
Redis connection "redis://localhost/2" failed: Redis server went away. Revert to 6.2.2 and everything works again.

When i revert the change from row 222 of https://github.com/symfony/symfony/pull/48711:

**Comment by powernic** [comment, marked as answer]
Best regards

Your fan
